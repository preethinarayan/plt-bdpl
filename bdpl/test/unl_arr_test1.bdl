bit[*] a (fieldsize 20);
file "./test.dat" inp_file;
print("finished reading input file ... creating array");
read(inp_file,a);
print(a);
