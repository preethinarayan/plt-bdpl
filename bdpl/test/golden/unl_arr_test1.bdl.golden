finished reading input file ... creating arrayARRAY of TYPE_BIT
[
	0 => 0,
	1 => 1,
	2 => 1,
	3 => 0,
	4 => 0,
	5 => 0,
	6 => 0,
	7 => 1,
	8 => 0,
	9 => 1,
	10 => 1,
	11 => 0,
	12 => 0,
	13 => 0,
	14 => 1,
	15 => 0,
	16 => 0,
	17 => 1,
	18 => 1,
	19 => 0,
]
